### ISSUE TITLE
Total Budget Optimizing Result has incorrect initial total response

### ISSUE THREAD

**Issue by rohan1997vk** [facebookexperimental/Robyn#919]
## Describe issue
The total response for the last 4 weeks is ~750K. However, after a 2000ierations with 5 trails and selecting a model, the budget allocator shows that the total response is ~350K. The actual spend is 1.06M and the actual revenue is ~750K. R2 is around 0.90

## Provide reproducible example
![image](https://github.com/facebookexperimental/Robyn/assets/22797824/ec3e84a8-498f-48b1-b05c-3f5b0e859e8d)

## Environment & Robyn version
Make sure you're using the latest Robyn version before you post an issue.
> packageVersion("Robyn")
[1] ‘3.10.3’
> sessionInfo()
R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 11 x64 (build 22631)

**Comment by gufengzhou** [comment]
The total response refers to media only response, or revenue contribution from media, NOT the total revenue. The budget allocator only maximises the media contribution and it can't move anything for trend/season/others that have no spend. Makes sense?

**Comment by apoorvabc** [comment]
Just curious, Does this mean the bounded response output also contains the spend and revenue of media channels alone and doesnt include the trend/season/competitors and other factors? Do we have to add these values up incase we are looking for the total revenue earned during the time period?

**Comment by gufengzhou** [comment]
Yes and that'd imply forecast of trend/season etc that is quite a topic of itself. But yes you're right.

**Comment by AlainTiri** [comment]
Hello @gufengzhou, it is very clear. However, I would like to bring to your attention that it is problematic to present a graph with these figures without explicit documentation. In my case, I have 22k conversions, but the reallocation graph mentions initResponseTotal:+-7k. (My rmse is very bad). A client will want to see the... ResponseTotal no matter what. Same for the calculation of delta and totalLift. It's frustrating  ;-) 

Kind regards.

**Comment by gufengzhou** [comment]
A model with bad rmse shouldn't be interpreted in the first place. Please reopen if necessary.
